tools/packaging/debian/precice.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=${prefix}
+includedir=${prefix}/include
```

Comment:
This should be:

```suggestion
includedir=${prefix}/include/precice
```

Reply:
This should be `prefix/include` as we install our interface headers as `prefix/include/precice/[C]SolverInterface.h[pp]`.

Clients _should_ always use the scoped includes `#include <precice/SolverInterface.h[pp]>` to prevent conflicts with other libraries. (also part of xSDK)